wolfhsm/wh_server.h

```diff
@@ -218,6 +218,11 @@ struct whServerContext_t {
     uint8_t padding[4];
 };
 
+#define WH_COMMSERVER_INITIALIZER \
+{ .packet = {0} }
+#define WH_SERVER_CONTEXT_INITIALIZER \
+{ .comm = {WH_COMMSERVER_INITIALIZER} }
+
```

Comment:
Random thought: I'm wondering if it is okay to violate style conventions (in this case, always using `SCREAMING_SNAKE_CASE` for macros) in order to make the struct name perfectly explicit in the macro. 

`whServerContext_INITIALIZER`, for example

just a thought

Reply:
Reordered the structs and removed the initializer.  OBE.